### PR TITLE
Adding dcc

### PR DESCRIPTION
**Pull request by SeanDS**
This adds [dcc](https://docs.ligo.org/sean-leavey/dcc/), a CLI for the LIGO Document Control Center.

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by SeanDS** [comment]
@duncanmmacleod are you happy to be a co-maintainer of this?

**Comment by duncanmmacleod** [comment]
> @duncanmmacleod are you happy to be a co-maintainer of this?

Yes, I'm happy to help maintain this.

**Comment by duncanmmacleod** [comment]
@conda-forge/staged-recipes, this is ready for review, thanks.

File: recipes/dcc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    - dcc scrape --help
+    - dcc update --help
+    - dcc view --help
+
+about:
+  home: https://docs.ligo.org/sean-leavey/dcc/
+  dev_url: https://git.ligo.org/sean-leavey/dcc/
+  summary: Tools for interactive and programmatic access to the LIGO DCC.
+  license: GPL-3.0-or-later
+  license_family: GPL
```
Comment (dopplershift) [inline recipes/dcc/meta.yaml:63]:
```suggestion
  license_family: GPL3
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
Minor tweak to the license metadata and this is good to go.

**Comment by SeanDS** [comment]
@conda-forge/staged-recipes, this is ready for review again.

**Review by dopplershift — APPROVED** [review]

**Pull request merged**